public/js/contribute/pad/load.js

```diff
@@ -69,22 +68,24 @@ async function DOMLoad() {
       });
 
       const url = new URL(window.location);
-      if (!queryparams) var queryparams = new URLSearchParams(url.search);
-      queryparams.delete('display');
+      if (!window.queryparams)
```

Comment:
it's probably better to not make it a global especially since we're manipulating it (deleting display for example -- this will happen every time without resetting the search params)